Explorer/Assets/DCL/Rendering/GPUInstancing/GPUInstancingRenderFeature.GPUInstancingRenderPass.cs

```diff
@@ -0,0 +1,62 @@
+﻿using DCL.Roads.GPUInstancing;
+using ECS;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public partial class GPUInstancingRenderFeature
+{
+    public class GPUInstancingRenderPass : ScriptableRenderPass
+    {
+        private const string PROFILER_TAG = "_DCL.GPUInstancingRenderPass";
+
+        private GPUInstancingService instancingService;
+        private IRealmData realmData;
+
+        public GPUInstancingRenderPass(GPUInstancingService service)
+        {
+            this.instancingService = service;
+        }
+
+        public void SetService(GPUInstancingService service, IRealmData realmData)
+        {
+            instancingService = service;
+            this.realmData = realmData;
+        }
+
+        // Called before executing the render pass.
+
+        public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderingData)
+        {
+            // Configure camera targets, if needed
+        }
+
+        // The actual execution of the pass
+
+        public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
+        {
+            return;
```

Comment:
I guess this code is still not ready for review?

Reply:
I removed that `return`. However, currently this render feature is disabled. It is for our next move in the direction of going from `Graphics API` to `CommandBuffer` approach